### PR TITLE
removed extraneous 'RDS'

### PR DESCRIPTION
**Pull request by 0atman**
-------

What
----

there was an extra 'rds' on the masketplace page.

How to review
-------------

read it

Who can review
---------------

people with eyes

---

🚨⚠️ Please do not merge this pull request via the GitHub UI ⚠️🚨

### CODE REVIEW

**Review by paroxp — CHANGES_REQUESTED** [review]
This will also change `AWS RDS Postgres` to `AWS Postgres` which is strictly speaking misinformation as it could also be `AWS Aurora Postgres`.

File: src/components/marketplace/views.tsx [hunk 1]
```diff
@@ -159,7 +159,7 @@ const serviceDetails: IServiceDetails = {
   'postgres': {
     image: postgresLogo,
     imageTitle: 'PostgreSQL - Official Logo',
-    name: 'Postgres',
+    name: 'RDS Postgres',
```
Comment (0atman) [inline src/components/marketplace/views.tsx:162]: @paroxp how's that?

Comment (paroxp) [reply src/components/marketplace/views.tsx:162]: Add the same for MySQL and we have a deal.

Comment (0atman) [reply src/components/marketplace/views.tsx:162]: BEHOLD!

**Review by paroxp — APPROVED** [review]

**Pull request merged**